<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DrumWidget</class>
 <widget class="QWidget" name="DrumWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Magnetic drum (RAM)</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Representation</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="octSwitch">
        <property name="text">
         <string>Octal</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">formatGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="binSwitch">
        <property name="text">
         <string>Binary</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">formatGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="clearButton">
     <property name="text">
      <string>Clear</string>
     </property>
     <property name="icon">
      <iconset resource="../libural/images.qrc">
       <normaloff>:/images/edit-clear.png</normaloff>:/images/edit-clear.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTableWidget" name="drumView">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <column>
        <property name="text">
         <string>00</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>01</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QScrollBar" name="drumScroll">
       <property name="maximum">
        <number>2046</number>
       </property>
       <property name="singleStep">
        <number>2</number>
       </property>
       <property name="pageStep">
        <number>16</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../libural/images.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>on_drumScroll_valueChanged(int)</slot>
 </slots>
 <buttongroups>
  <buttongroup name="formatGroup"/>
 </buttongroups>
</ui>
